### PR TITLE
Added 6 new events to the profile

### PR DESCRIPTION
**Pull request by imran1509**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4647"></a>

### CODE REVIEW

**Comment by imran1509** [comment]
Why is this giving syntax error. I have copied it from official docs and replace my details

File: data/imran1509/events/2023-05-17-talk-kubehuddle.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "isInPerson": true,
+  "color": "blue",
+  "name": "KubeHuddle Toronto 2023",
+  "description": "KubeHuddle is a community conference where Developers, Platform Engineers, DevOps, SRE, Cloud Enthusiasts, Technical and Business Strategists come together to learn from each other, collaborate and, innovate.",
+  "date": {
+    "start": "2023-05-17T10:00:00.000+00:00",
+    "end": "2023-05-18T18:00:00.000+00:00",
+    "cfpClose": "2023-03-03T11:59:00.000+00:00"
+  },
+  "https://kubehuddle.com/2023/toronto/"
```
Comment (Panquesito7) [inline data/imran1509/events/2023-05-17-talk-kubehuddle.json:12]: The URL field is missing here.
```suggestion
  "url": "https://kubehuddle.com/2023/toronto/"
```

File: data/imran1509/events/2023-06-08-talk-Platformcon.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "isInPerson": true,
+  "color": "blue",
+  "name": "PlatformCon 2023",
+  "description": "he brightest Platform and DevOps pioneers, on one stage 🚀 PlatformCon was born to celebrate our community of 10k+ Platform Engineers. The response to the first-ever edition was overwhelming. PlatformCon 2023 is already making us dream big.",
+  "date": {
+    "start": "2023-06-08T10:00:00.000+00:00",
+    "end": "2023-06-09T18:00:00.000+00:00",
+    "cfpClose": "2023-02-28T18:00:00.000+00:00"
+  },
+  "https://platformcon.com/"
```
Comment (Panquesito7) [inline data/imran1509/events/2023-06-08-talk-Platformcon.json:12]: Same here.
```suggestion
  "url": "https://platformcon.com/"
```

File: data/imran1509/events/2023-06-15-talk-DevOpsDaysBirmingham.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "isInPerson": true,
+  "color": "blue",
+  "name": "DevOpsDays Birmingham (UK) 2023",
+  "description": "DevOpsDays is a worldwide series of community run technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. It is run by volunteers from community, for the benefit of the community. We are not a commercial conference and we believe that our focus on serving the community creates a truly unique experience for both delegates and sponsors.",
+  "date": {
+    "start": "2023-06-15T10:00:00.000+00:00",
+    "end": "2023-06-16T18:00:00.000+00:00",
+    "cfpClose": "2023-02-24T18:00:00.000+00:00"
+  },
+  "https://devopsdays.org/events/2023-birmingham-uk/welcome/"
```
Comment (Panquesito7) [inline data/imran1509/events/2023-06-15-talk-DevOpsDaysBirmingham.json:12]:
```suggestion
  "url": "https://devopsdays.org/events/2023-birmingham-uk/welcome/"
```

File: data/imran1509/events/2023-09-20-talk-SwissCloudNativeDay.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  "isInPerson": true,
+  "color": "blue",
+  "name": "Swiss Cloud Native Day 2023",
+  "description": "The local Cloud Native community is organizing the third edition of the Swiss Cloud Native Day. Our event aims to bring newcomers and experts in the cloud-native space together and be an inspirational place where different perspectives and experiences meet.",
+  "date": {
+    "start": "2023-09-20T10:00:00.000+00:00",
+    "end": "2023-09-21T18:00:00.000+00:00",
+    "cfpClose": "2023-04-30T18:00:00.000+00:00"
+  },
+  "https://cloudnativeday.ch/"
```
Comment (Panquesito7) [inline data/imran1509/events/2023-09-20-talk-SwissCloudNativeDay.json:12]:
```suggestion
  "url": "https://cloudnativeday.ch/"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**